### ISSUE TITLE
Add extension compatibility to Opera GX

### ISSUE THREAD

**Issue by Kuroyuki-Kaze** [doki-theme/doki-theme-web#103]
**Is your feature request related to a problem? Please describe.**
Not a problem or anything, I just would like to use this extension with Opera GX. I primarily use GX for pretty much everything so I thought it would be very cool if I had the prebuilt anime themes too (particularly because I have no idea what color goes where).

**Describe the solution you'd like**
A simple extension load just like all the other ones like Chrome.

**Describe alternatives you've considered**
I don't really have any idea on how to load the extensions or somehow do CSS magic on the browser, I'm not that at that level yet, sorry.

**Comment by Unthrottled** [comment]
Yeah, when I get time I'll look into it. Looks like it is chrome based because you can go to `chrome://extensions` and add extensions that way. However, It appears opera has different extension support, because the theme & newTab extensions don't work.

**Comment by PixelHir** [comment]
Opera GX has their own system for themes and theme store
![image](https://user-images.githubusercontent.com/7670940/212151274-9c9b798b-daf4-4a83-bec0-7d0980442793.png)
Unfortunately it's basically limited to only setting a background image, that's it.
Contents of Opera GX theme:
![image](https://user-images.githubusercontent.com/7670940/212151431-1db76ed4-486f-419e-a0bb-0888f5411830.png)
